### ISSUE TITLE
Three questions

### ISSUE THREAD

**Issue by sergei-bondarenko** [ZhengyaoJiang/PGPortfolio#125]
1. Why are you publicly shared the algorithm if the results are so good and consistent?

2. In readme you wrote "Original versioning history, and internal discussions, including some in-code comments, are removed in this open-sourced edition. These contains our unimplemented ideas, some of which will very likely become the foundations of our future publications". In other issues you also refused to give any even smallest hints about these ideas. Why are you behaving like that?

3. Did you run test only once just to measure results? Or you run it multiple times to tweak the model? I know that you used CV for hyperparameters tuning, but there is also can be a data leakage if you run your model on a test set and tweak it a little bit if the results are bad.

Hope you will answer. Thank you!

**Comment by liuyangdh** [comment]
I reimplemented their work, the model can perfectly fit the training data, but can NOT generalize to unseen data robustly. By "not robust" I mean,
1. different seed yields very different results
2. even though for one model it can outperform in some chuck of unseen data, but on other data it behaves very badly

either I missed something from their paper and implementation or I have to conclude that this work is NOT practical at all.

**Comment by tr8dr** [comment]
@liuyangdh Do you have your reimplementation available?  Am curious to see what you did.   Thx!
